src/pipeline.jl

```diff
@@ -1,25 +1,56 @@
 ## Stubs
 function _recipe_init!(plt, plotattributes, args) end
 function _recipe_after_user!(plt, plotattributes, args) end
-function _recipe_after_plot!(plt, plotattributes, args) end
-function _recipe_after_series!(plt, plotattributes, args, series_ind) end
```

Comment:
I'd still like to also have the `after_series` hook.

Reply:
yeah, there is an inconsistency here. This is due to the fact that I didn't know if or how to change the order of operations here: https://github.com/JuliaPlots/Plots.jl/blob/20a290792f92b3f53b8e5ba91ccfb2cdd8759d0d/src/plot.jl#L218-L234

Are you suggesting to have both before and after or only after? The before part was to avoid the Plots.jl specific handling.

I'm also not sure I understand what happens in this part of the code, so I'd appreciate if anyone can clarify.

Reply:
I think having both is fine. Technically the before version is not necessary, but it is quite convenient.